.github/workflows/validate-retrieve-image-tags-config.yml

```diff
@@ -2,7 +2,7 @@ name: Validate retrieve-image-tags config
 on:
   pull_request:
     paths:
-      - 'retrieve-image-tags/config.json'
+      - 'retrieve-image-tags/**'
```

Comment:
Added this as it wasn't running the validate workflow when anything else than `config.json` was changed